src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/PagingDto.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Equinor.ProCoSys.IPO.Query.GetInvitations;
```

Comment:
remove

Reply:
remove the file ?

Reply:
wasn't it a unused using?

Reply:
Don't see any unused usings now at least